contracts/rust/deployer/src/builder.rs

```diff
@@ -284,12 +284,11 @@ impl<P: Provider + WalletProvider> DeployerArgs<P> {
                 let epoch_start_block = self.epoch_start_block.unwrap();
                 let rpc_url = self.rpc_url.clone();
 
-                // TEST-ONLY: if this config is not yet set, we use a large default value
+                // TEST-ONLY: if this config is not yet set, we use a reasonable default value
                 // to avoid contract complaining about invalid zero-valued blocks_per_epoch.
-                // This large value will act as if we are always in epoch 1, which won't conflict
-                // with the effective purpose of the real `PublicNetworkConfig`.
+                // This value will allow tests to proceed with realistic epoch behavior.
                 if use_mock && blocks_per_epoch == 0 {
-                    blocks_per_epoch = u64::MAX;
+                    blocks_per_epoch = 10;
```

Comment:
@alxiong do you remember why we set `u64::max` here? I recall there was some kind of hack for the prover to make sure we stay in epoch 0.

cc @alysiahuggins

Reply:
I don't think we should need to set this in a real env. @mrain @alxiong?

Reply:
ok so you're saying that we want to allow the setting of the `exitEscrowPeriod` to u64::max / uint64.max (during initialization) to prevent epoch changes in the contract? Sure, can revert.

The max exitEscrowPeriod is also limited in the `updateExitEscrowPeriod()` method, I think that makes sense because otherwise that can halt epoch transitions (and i think there are other ways to halt withdrawals etc - example via pausing which is already implemented and a lot clearer as a governance operation). 

https://github.com/EspressoSystems/espresso-network/blob/536ad560aefd9d8b42e6033686fe817b3036486d/contracts/src/StakeTableV2.sol#L182-L193